ccd-definition/CaseEventToComplexTypes/messageJudge/judicialMessageMetaData.json

```diff
@@ -4,27 +4,48 @@
     "ID": "JudicialMessageMetaData",
     "CaseEventID": "messageJudgeOrLegalAdviser",
     "CaseFieldID": "judicialMessageMetaData",
-    "ListElementCode": "sender",
+    "ListElementCode": "senderType",
     "FieldDisplayOrder": 1,
-    "DisplayContext": "MANDATORY"
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "isJudiciary!=\"YES\""
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "JudicialMessageMetaData",
     "CaseEventID": "messageJudgeOrLegalAdviser",
     "CaseFieldID": "judicialMessageMetaData",
-    "ListElementCode": "recipient",
+    "ListElementCode": "sender",
     "FieldDisplayOrder": 2,
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "judicialMessageMetaData.senderType=\"LOCAL_COURT_ADMIN\" OR judicialMessageMetaData.senderType=\"OTHER\""
```

Comment:
Do we need this because we don't want to show the address field for the CTSC?

Reply:
Yes.